tests/test_unit.py

```diff
@@ -0,0 +1,48 @@
+from docfx_yaml.extension import find_unique_name
+from docfx_yaml.extension import disambiguate_toc_name
+
+import unittest
+
+from yaml import load, Loader
+
+class TestGenerate(unittest.TestCase):
+    def test_find_unique_name(self):
+
+        entries = {}
+        
+        # Disambiguate with unique entries.
+        entry1_1 = "google.cloud.aiplatform.v1.schema.predict.instance_v1.types"
+        entry1_2 = "google.cloud.aiplatform.v1beta2.schema.predict.instance_v1.types"
+        want1_1 = "v1.types"
+        want1_2 = "v1beta2.types"
```

Comment:
Why the `1`? (e.g. entry**1**_1).

Reply:
I had few more sets of tests, then realized one will be good enough to test but forgot to remove the grouping prefix. Removed it.